### ISSUE TITLE
When a block theme is active, provide a link to edit the page template from the pages list

### ISSUE THREAD

**Issue by jameskoster** [WordPress/gutenberg#41716]
On the Pages list in wp-admin, it would be nice to offer a shortcut to edit the default page template.

**Comment by LukaszJaro** [comment]
Also the ability to block this from some users.

**Comment by jameskoster** [comment]
@LukaszJaro it would be governed by the native roles and capabilities. Iirc only admins are able to edit the template. Did you have something more specific in mind?

**Comment by draganescu** [comment]
@jameskoster is this more than simply adding the link like below:

![Frame 1](https://github.com/WordPress/gutenberg/assets/107534/1a81d4a5-5541-47c9-93e0-6e2a85f4a652)

Next to Search results and Page 404 we can add a Single page link to the page template.

**Comment by jameskoster** [comment]
@draganescu that's correct. But this will probably need a fresh design given the ambition is to replace this panel with the data view (List layout).

**Comment by draganescu** [comment]
So should we wait then?

**Comment by jameskoster** [comment]
I think so. We'll need to figure out how to link to these templates from the pages data views. They could theoretically appear amongst regular pages with special marking to denote their significance. But that gets tricky because we'd be mixing record types (pages + templates) which adds complexity to filtering, view options, etc. Ideas welcome if you have any!

**Comment by richtabor** [comment]
> But that gets tricky because we'd be mixing record types (pages + templates) which adds complexity to filtering, view options, etc.

Agreed. I'm concerned about making it less clear what is template and what is page.

**Comment by jameskoster** [comment]
One option could be a 'Utility' pseudo-view within the page management section. It would list all templates conceptually associated with pages.

<img width="355" alt="Screenshot 2024-03-19 at 10 41 27" src="https://github.com/WordPress/gutenberg/assets/846565/5b6e5948-74f4-4754-b7ed-029ba4ca310e">
